sdk/core/Azure.Core.Amqp/Azure.Core.Amqp.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Not sure why we didn't have a solution previously;  please let me know if there was a reason and I can pull this back out.